public/components/page-contributors/contributors/contributors.stache

```diff
@@ -10,24 +10,24 @@
     </tr>
   </thead>
   <tbody>
-  {{#each contributors as contributor}}
+  {{# each(contributors, contributor=value) }}
     <tr>
-      <td><model-edit-property model:from="contributor" property="name" /></td>
-      <td><model-edit-property model:from="contributor" property="email" /></td>
+      <td><model-edit-property model:from="contributor" property:from="'name'" /></td>
+      <td><model-edit-property model:from="contributor" property:From="'email'" /></td>
```

Comment:
Any reason for uppercase `:From`?

Reply:
This was a mistake.  Thanks for spotting it!